types/src/on_chain_config/aptos_features.rs

```diff
@@ -76,17 +129,9 @@ impl Default for Features {
             features: vec![0; 5],
         };
 
-        use FeatureFlag::*;
```

Comment:
Where is this used in default_if_missing? I don't see that.  We actually want to make it that way (i.e. move unit tests run with all flags disabled at the moment), but seems like we don't have that anywhere as far as I can see

Reply:
nvm, my impression was from probably randomness branch 😅

Because we introduced a FeatureFlag for randomness, many component now needs the `Features` resource to be loaded on new epoch (e.g., StateComputer), where a default value is needed in case of no `Features` on chain, and that default value should not be `default_for_genesis`. (but it's probably safe to assert that `Features` resource can be loaded?)

Reply:
yes, I think it is safer to expect Features to exist